packages/ui/src/styles/globals.css

```
@@ -11,6 +12,9 @@
 /* AI Elements' MessageResponse/Reasoning render markdown via Streamdown, whose
  * prose/code classes live in its dist JS — scan them so they aren't purged. */
 @source "../../node_modules/streamdown/dist/*.js";
+@source "../../node_modules/@streamdown/code/dist/*.js";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)